src/hypertrace/agent/instrumentation/flask/__init__.py

```diff
@@ -58,11 +61,24 @@ def hypertrace_before_request() -> None:
             request_body = flask.request.data       # same
             logger.debug('span: %s', str(span))
             logger.debug('Request headers: %s', str(request_headers))
+
             span.update_name(str(flask.request.method) + ' ' + str(flask.request.url_rule))
 
             # Call base request handler
             flask_wrapper.generic_request_handler(
                 request_headers, request_body, span)
+
+            block_result = Registry().apply_filters(span,
+                                                    flask.request.url,
+                                                    flask.request.headers,
+                                                    flask.request.data)
+            if block_result:
```

Comment:
on blocking do we add some attribute in a span that the request was blocked by traceable?

Reply:
I think that would happen within the individual filter, not necessarily here.  

I'm basing my answer on javaagent filter:
https://github.com/hypertrace/javaagent/tree/6b787b4a62ff41cfbd042b48f16619b36e4f7a32/filter-api/src/main/java/org/hypertrace/agent/filter

The files within that package seem to be around registration and applying filters, not adding span attributes.

Reply:
ah. i see

Reply:
Yeah, the traceable implementation of the java filter API adds span attributes, but i agree it makes to not add those attributes if there's no filter included